docs/includes/generated_docs/schemas/beta.core.md

```diff
@@ -124,7 +135,7 @@ The `ehrql.tables.beta.raw.ons_deaths` table contains all registered deaths.
     <code>date</code>
   </dt>
   <dd markdown="block">
-Patient's date of death. Only deaths registered from February 2019 are recorded.
```

Comment:
I thought it would be better to highlight this in the main documentation so it's easier to see.